.github/workflows/e2e-testing.js.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Cypress Tests
+
+on:
+  push:
+
+    branches: ["workflow"]
+  pull_request:
+    branches: ["workflow"]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
"Why are you running your tests on a deprecated version of Node?". Only need to run on lts version